temporalcli/commandsmd/commands.md

```diff
@@ -675,677 +1329,1409 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
-* `--ui-codec-endpoint` (string) - UI remote codec HTTP endpoint.
-* `--sqlite-pragma` (string[]) - Specify SQLite pragma statements in pragma=value format.
-* `--dynamic-config-value` (string[]) - Dynamic config value, as KEY=JSON_VALUE (string values need quotes).
-* `--log-config` (bool) - Log the server config being used to stderr.
-* `--search-attribute` (string[]) - Search attributes to register, in key=type format.
+* `--db-filename`, `-f` (string) -
+  Path to file for persistent Temporal state store.
+  By default, Workflow Executions are lost when the server process dies.
+* `--namespace`, `-n` (string[]) -
+  Namespaces to be created at launch.
+  The "default" Namespace is always created automatically.
+* `--port`, `-p` (int) -
+  Port for the frontend gRPC Service.
+  Default: 7233.
+* `--http-port` (int) -
+  Port for the HTTP API service.
+  Default is off.
+* `--metrics-port` (int) -
+  Port for '/metrics'.
+  Default is off.
+* `--ui-port` (int) -
+  Port for the Web UI.
+  Default is '--port' value + 1000.
+* `--headless` (bool) -
+  Disable the Web UI.
+* `--ip` (string) -
+  IP address bound to the frontend Service.
+  Default: localhost.
+* `--ui-ip` (string) -
+  IP address bound to the WebUI.
+  Default is same as '--ip' value.
+* `--ui-public-path` (string) -
+  The public base path for the Web UI.
+  Default is `/`.
+* `--ui-asset-path` (string) -
+  UI custom assets path.
+* `--ui-codec-endpoint` (string) -
+  UI remote codec HTTP endpoint.
+* `--sqlite-pragma` (string[]) -
+  SQLite pragma statements in "PRAGMA=VALUE" format.
+* `--dynamic-config-value` (string[]) -
+  Dynamic configuration value using `KEY=VALUE` pairs.
+  Keys must be identifiers, and values must be JSON values.
+  For example: 'YourKey="YourString"'.
+  Can be passed multiple times.
+* `--log-config` (bool) -
+  Log the server config to stderr.
+* `--search-attribute` (string[]) -
+  Search attributes to register using `KEY=VALUE` pairs.
+  Keys must be identifiers, and values must be the search
+  attribute type, which is one of the following:
+  Text, Keyword, Int, Double, Bool, Datetime, KeywordList.
+
+### temporal task-queue: Manage Task Queues
+
+Inspect and update Task Queues, the queues that Workers poll for Workflow and
+Activity tasks:
+
+```
+temporal task-queue [command] [command options] \
+    --task-queue YourTaskQueue
+```
 
-### temporal task-queue: Manage Task Queues.
+For example:
 
-Task Queue commands allow operations to be performed on [Task Queues](/concepts/what-is-a-task-queue). To run a Task
-Queue command, run `temporal task-queue [command] [command options]`.
+```
+temporal task-queue describe \
+    --task-queue YourTaskQueue
+```
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal task-queue describe: Provides pollers information, backlog statistics, and task reachability (experimental) for Workers on this Task Queue.
+### temporal task-queue describe: Show active Workers
```

Comment:
The biggest conflict was the help text for this command. Everything else was a pretty straightforward merge.